projects/sound-open-firmware/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Please use the OSS-Fuzz apache license header in Dockerfile and build.sh

Missing license header in file /home/runner/work/oss-fuzz/oss-fuzz/projects/sound-open-firmware/Dockerfile.
Missing license header in file /home/runner/work/oss-fuzz/oss-fuzz/projects/sound-open-firmware/build.sh.
Failed.